Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -279,6 +279,17 @@ final class BolusEntryViewModel: ObservableObject {
         )
     }
 
+    private func presentAlert(_ alert: Alert) {
+        dispatchPrecondition(condition: .onQueue(.main))
+
+        // As of iOS 13.6 / Xcode 11.6, swapping out an alert while one is active crashes SwiftUI.
+        guard activeAlert == nil else {
```

Comment:
oy...this is only in 13.6? :\

Reply:
Probably prior as well, I left the comment just to note "this was the latest version I've checked on"